spec/models/article_spec.rb

```diff
@@ -18,4 +18,19 @@
     expect(Article.first.desc_facet).to eq("desc_facet")
     expect(Article.first.geo_facet).to eq("geo_facet")
   end
+
+  it {should have_many(:tweets)}
+
+ xit "should build associated tweets from article url" do
+     article_data = [{
+      title: "title",
+      url: "http://example.com",
+      abstract: "abstract",
+      desc_facet: "desc_facet",
+      geo_facet: "geo_facet"
+    }]
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.